Automattic/jetpack
# My Jetpack: consider Backup active only when plan is purchased

Fixes #23148

#### Changes proposed in this Pull Request:

* Only consider backup product active when the purchase was made

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:







* Install/activate the Jetpack plugin only
* Click to connect
* When you see the user connection step, click "back". This will put you in site-only connection state.
* Navigate to My Jetpack
* Make sure Backup is still inactive
* Click Add Backup
* Make sure the interstitial shows up
* Click Add Backup
* You should be redirected to Jetpack Dashboard (this flow will be improved in future PRs)
